### ISSUE TITLE
nothing to distinguish generators from functions

### ISSUE THREAD

**Issue by 75lb** [jsdoc3/jsdoc#1158]
Without writing "generator" in the text somewhere it's impossible to reliably determine whether a doclet (in the `--explain` data or generated output) is a function or generator. 

There is nothing to distinguish this function:

```
/**
 * a global function
 */
function globalFunction () {}
```

from this generator:

```
/**
 * a global generator
 */
function * globalGenerator () {}
```

**Comment by julmot** [comment]
:+1:

**Comment by hegemonic** [comment]
Fixed on master. The fix will be included in JSDoc 3.5.0.
